Automattic/wp-calypso
# Activity Log: Remove credentials footer

Since it was fairly redundant, let's remove the current footer. We should still add one to the credentials form view itself as shown in https://github.com/Automattic/wp-calypso/issues/20116

#### To test
Go to a site that hasn't had credentials entered and make sure the footer is no longer there.

#### Before
![image](https://user-images.githubusercontent.com/1123119/33141945-00fab150-cf72-11e7-9565-256657df02e6.png)

#### After
![image](https://user-images.githubusercontent.com/1123119/33141875-c59bab46-cf71-11e7-883e-59d67e424205.png)
